sphinx_bootstrap_theme/bootstrap/static/bootstrap-sphinx.js_t

```diff
@@ -97,7 +97,7 @@
 
       // Enlarge content if sidebar is larger.
       if ($sideBar.outerHeight(true) > $content.outerHeight(true)) {
-        $content.minHeight($sideBar.outerHeight(true));
+        $content.css('min-height', $sideBar.outerHeight(true));
```

Comment:
Can you switch to double quotes (which is the style used throughout the rest of the file).